config/mclag.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/sbin/env python
+
+import click
+import swsssdk
```

Comment:
we should use swss-common, no more swsssdk.

Reply:
@lguohan 
i have taken care of this comment. thanks